docker/serverless/env_start_proxy.py

```diff
@@ -26,6 +26,7 @@
  
  https://cloud.google.com/endpoints/docs/openapi/get-started-cloud-run#configure_esp
```

Comment:
Good point.

We only want to reference serverless docs here, not all docs in general. This was added because users were having trouble with step 2 of the serverless quick starts.

If we do end up adding more docs in the future, we can replace these links as you said to prevent growing the size.